src/app/_helpers/interceptors/login.interceptor.ts

```diff
@@ -94,4 +71,19 @@ export class LoginInterceptor implements HttpInterceptor {
       }
     }
   }
+
+  sendResponse(responseBody: {}, status: number): Observable<any> {
+    return new Observable<any>(
+      observer => {
+        observer.next(new HttpResponse<any>(
+          {
+            body: {
+              responseBody
+            },
+            status: status
+          }
+        ));
+        observer.complete();
+      });
+  }
```

Comment:
@HonzaDudek well, if we're using it in multiple interceptors we should put it into some utilities/helper for the interceptors